python-checks/src/main/java/org/sonar/python/checks/CommentRegularExpressionCheck.java

```diff
@@ -28,8 +27,7 @@
 
 @Rule(
   key = "CommentRegularExpression",
-  priority = Priority.MAJOR,
-  cardinality = Cardinality.MULTIPLE)
```

Comment:
Removing the cardinality makes the rule a simple rule instead of a rule template.